packages/datadog_flutter_plugin/doc/rum/advanced_configuration.md

```diff
@@ -121,6 +121,101 @@ For example:
+
+For example, to redact sensitive information in a RUM Resource's `url`, implement a custom `redacted` function and use it in `rumResourceEventMapper`:
+
+```dart
+    rumResourceEventMapper = (event) { 
+        var resourceEvent = resourceEvent
+        resourceEvent.resource.url = redacted(resourceEvent.resource.url)
+        return resourceEvent
+    }
+}
+```
+
+Returning `null` from the error, resource, or action mapper drops the event entirely; the event is not sent to Datadog. The value returned from the view event mapper must not be `null`.
+
+Depending on the event's type, only some specific properties can be modified:
+
+| Event Type       | Attribute key                     | Description                                   |
+|------------------|-----------------------------------|-----------------------------------------------|
+| RumViewEvent     | `viewEvent.view.name`             | Name of the view.                             |
+|                  | `viewEvent.view.url`              | URL of the view.                              |
+|                  | `viewEvent.view.referrer`         | Referrer of the view.                         |
+| RumActionEvent   | `actionEvent.action.target?.name` | Name of the action.                           |
+|                  | `actionEvent.view.name`           | Name the view linked to this action.          |
+|                  | `actionEvent.view.referrer`       | Referrer of the view linked to this action.   |
+|                  | `actionEvent.view.url`            | URL of the view linked to this action.        |
+| RumErrorEvent    | `errorEvent.error.message`        | Error message.                                |
+|                  | `errorEvent.error.stack`          | Stacktrace of the error.                      |
+|                  | `errorEvent.error.resource?.url`  | URL of the resource the error refers to.      |
+|                  | `errorEvent.view.name`            | Name the view linked to this action.          |
+|                  | `errorEvent.view.referrer`        | Referrer of the view linked to this action.   |
+|                  | `errorEvent.view.url`             | URL of the view linked to this error.         |
+| RumResourceEvent | `resourceEvent.resource.url`      | URL of the resource.                          |
+|                  | `resourceEvent.view.name`         | Name the view linked to this action.          |
+|                  | `resourceEvent.view.referrer`     | Referrer of the view linked to this action.   |
+|                  | `resourceEvent.view.url`          | URL of the view linked to this resource.      |
+
+## Set tracking consent (GDPR compliance)
+
+In order to be compliant with the GDPR regulation, the Flutter RUM SDK requires the tracking consent value at initialization.
```

Comment:
```suggestion
In order to be compliant with data protection and privacy policies, the Flutter RUM SDK requires the tracking consent value at initialization.
```

Reply:
We should probably go back and make similar adjustments in the iOS and Android documentation.